astropy/table/tests/test_mixin.py

```diff
@@ -204,8 +195,7 @@ def test_io_time_write_fits(tmpdir, table_types):
 def test_io_write_fail(mixin_cols):
```

Comment:
again, maybe change the name of the test?

Reply:
Done.